braindecode/datautil/serialization.py

```diff
@@ -23,24 +25,43 @@ def save_concat_dataset(path, concat_dataset, concat_of_raws, overwrite):
         directory to which .fif and .json files are stored
     concat_dataset: BaseConcatDataset of BaseDatasets or WindowsDatasets
         to save to files
-    concat_of_raws: bool
-        if true assumes that concat_dataset contains raws, if false assumes
-        that concat_dataset contains epochs
     overwrite: bool
-        whether to overwrite existing files
+        whether to overwrite existing files (will delete old fif files)
```

Comment:
how about updating the comment to "will delete existing fif files in specified directory"?